modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/events.js

```diff
@@ -175,7 +175,9 @@
 
 				form.attr('action', action);
 
-				Util.submitForm(form);
+				var formNode = form._node || form;
```

Comment:
Same as above, I think we can remove this check and only have `from.submit()` to replace the util

Reply:
Lack of this check is what caused all those regressions a couple weeks ago. @bryceosterhaus